### PR TITLE
Allow only one leaderboard refresh every 2 minutes (#73)

### PR DESCRIPTION
**Pull request by MatthieuBonne**
just to lighten server load during WE

### CODE REVIEW

**Comment by Simoris** [comment]
tbh I'm not sure how good this fix it. It'll bring some relieve to the issue, but might lead to a load of complains from people, who just claimed their SJ misssion and don't see themself moving up in the leaderboard...

**Comment by MatthieuBonne** [comment]
Maybe we can add a notice somewhere, "updated every 2 minutes" ?

File: client/js/game.js [hunk 1]
```diff
@@ -16054,7 +16054,9 @@ function Game() {
         } else T.draw(ctx,"cjq4",W*0.5+bgw*0.5-T.width("cjq4")*0.5-14,H*0.5-bgh*0.5,T.width("cjq4")*0.75,T.height("cjq4")*0.75); 
     }
     this.updateLoopEventRanking = function (leaderboardName) {
-        if (eventRanking==undefined) eventRanking={};
+        if (eventRanking==undefined) eventRanking={time: 0, ptime: 0};
+		if(Date.now() - eventRanking.time < 2*60*1000)
+			return false;
```
Comment (Simoris) [inline client/js/game.js:16059]: Still not sure about this bit, maybe remove it for now and look how much of a diffrence the other changes make?

```suggestion
```

Comment (MatthieuBonne) [reply client/js/game.js:16059]: Ok

**Pull request merged**